1_fetch.R

```diff
@@ -94,7 +94,9 @@ p1_targets_list <- list(
   tar_target(
     p1_daily_data,
     get_daily_nwis_data(p1_nwis_sites_daily, parameter, stat_cd_select,
-                        start_date = earliest_date, end_date = latest_date),
+                        start_date = earliest_date, end_date = latest_date) %>%
+      #add column to identify the data source as daily
+      mutate(data_type = 'd'),
```

Comment:
I view the `p2_daily_combined` as a workaround to our current use of never rebuilding this `p1` target. But it will work just the same to assign the data_type in `p2` and not here. I can delete this